css/components/Modal.css

```diff
@@ -0,0 +1,41 @@
+@import "../style.css";
+
+.modal{
+    background-color: var(--badge-offline);
+    width: 20rem;
+    margin: 1rem;
+    font-size: 1rem;
+}
+
+.title{
+    padding: .5rem;
+    display: flex;
+    justify-content: space-between;
+}
+
+.content{
+    padding: 1rem;
+}
+
+.btn{
+    display: flex;
```

Comment:
If flex property is used more often, You can move this to utility file.

Reply:
done